solr/solr-ref-guide/src/solr-plugins.adoc

```diff
@@ -17,8 +20,36 @@
 // specific language governing permissions and limitations
 // under the License.
 
-Solr allows you to load custom code to perform a variety of tasks within Solr, from custom Request Handlers to process your searches, to custom Analyzers and Token Filters for your text field. You can even load custom Field Types. These pieces of custom code are called _plugins_.
+One of Solr's strengths is providing a rich platform of functionality with the option of adding your own custom components running within Solr.
+Solr calls such components *plugins* when the implementation is configurable.
```

Comment:
All our HTML pages treat the opening paragraph of a page as the "lede" and CSS makes it larger. For this reason we should avoid large blocks and try to open the page with a sentence that summarizes the feature/function if possible (it can be hard). In this case, the first sentence is rather perfect, so I'd suggest adding a new line here to separate the lede from the next batch of information which starts to get into the details.